### DISCUSSION TITLE
购买了一年的专业版，邮箱收不到验证码，请帮忙解决一下

### DISCUSSION THREAD

**Discussion by ImmortalZHL** [getlantern/lantern-forum-cn#787] [Q&A]
我的注册邮箱：immortal.zhl@gmail.com
下图是我的付款记录
![018b0702b813582d2187e77972251cd](https://github.com/user-attachments/assets/992aa6cf-1842-402c-a91f-58be46393969)

**Comment by ImmortalZHL** [comment]
@Mike777Khoo麻烦帮忙处理一下

**Reply by MaxMKMax** [reply]
@ImmortalZHL 您好. 谢谢您提供的资料. 已经帮您提交问题上去了. 请放心. 谢谢您支持蓝灯!

**Comment by MaxMKMax** [comment]
@ImmortalZHL 
您好! 这是您的验证码:
83Y79-XFPVD-QWV83-6TGR7-PJXBG
